### PR TITLE
Added frontend sum function

### LINKED ISSUE #26144: sum

### PR DESCRIPTION
**Pull request by gbit1605**
# PR Description
build: added the sum function to paddle frontend

## Related Issue

Close #26144

## Checklist

- [ Yes ] Did you add a function?
- [ Yes ] Did you add the tests?
- [ No ] Did you run your tests and are your tests passing?
- [ No ] Did pre-commit not fail on any check?
- [ Yes ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by gbit1605** [comment]
@bipinKrishnan Please let me know if this function can be merged into the main branch or if anything else needs to be done from my end.

**Comment by bipinKrishnan** [comment]
Hey @gbit1605, can you correctly link the issue you are working on by tagging it in the PR descript like "Close #<issue id>"?

**Comment by gbit1605** [comment]
> Hey @gbit1605, can you correctly link the issue you are working on by tagging it in the PR descript like "Close #"?

@bipinKrishnan Sure, added.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -317,6 +317,37 @@ def test_paddle_is_floating_point(
...
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+):
+    input_dtype, x = dtypes_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={"data": x[0]},
```
Comment (bipinKrishnan) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:340]: Tests are failing due to missing `backend_to_test` argument.

Comment (gbit1605) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:340]: Added the `backend_to_test` argument to the sum test function.

**Comment by gbit1605** [comment]
@bipinKrishnan is this ready to be merged into the main branch now?

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py (hunk 1 above)
Comment (bipinKrishnan) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:340]: Hey, I can't see the `backend_to_test` argument added anywhere, the tests are still failing due to this reason.

**Pull request closed without merging**